### ISSUE TITLE
Joomla Accessibility Statement

### ISSUE THREAD

**Issue by zwiastunsw** [joomla/accessibility#12]
What next steps?

**Comment by armenos** [comment]
I think we can put this version online now
And afterwards everyone can give his opinion for making it better.
Joomla! Extensions part must be excluded for now


## Joomla Accessibility Statement
Joomla! means “all together”. Inclusion is in our heart. As a community, we do our best to accept and welcome everyone. As such, we are committed to being accessible to the widest possible audience, regardless of ability or technology.

There are two main areas where we are focusing our efforts: the Joomla! CMS and Framework as our “product” and our websites - www.joomla.org and all of our subdomain sites.

### Joomla! CMS, Framework and extensions
Joomla! should be an accessible tool and provide a solid out of the box base for building accessible websites.

Since the birth of Joomla we have made efforts to make our software meet accessibility requirements. 

With Joomla! 4.0, our next milestone, we are determined to push further down the road to accessibility. With this in mind, we will strive to ensure full accessibility of all modern dynamic user interface elements. We will, first and foremost, ensure full accessibility of Joomla’s backend so that those with disabilities can use Joomla to create, administer and maintain sites in barrier-free way.

### We will give you the tools and the rest is up to you!
While Joomla core enables the creation of accessible AA-level sites,  it is up to those building individual sites to implement their own front end implementations in an accessible way. To put it another way, we will give you an accessible base install, but it will be up to you to ensure it is still accessible at the end of the build. 
### Joomla! extensions (this part needs to be discussed jet)
Core Joomla extensions meet WCAG 2.0 standards to at least AA level. However, many site developers use third-party extensions from the Joomla! Extensions Directory. Starting with Joomla! 4.0, authors of all extensions provided in the Directory will declare their level of accessibility. We recommend to check for accessibility before you purchase or install an extension and we encourage all authors to build their extensions with accessibility in mind.

### Joomla.org website and subdomains
We are aware that our own websites do not currently meet accessibility guidelines. We are actively working to fix this, and aim for our properties to also meet WCAG 2.0 AA standard when our new global template is rolled out alongside Joomla! 4. We will update this statement when this has been achieved.

### Joomla! Accessibility Team (JAT)
Accessibility, much like security, is a constantly evolving journey.The accessibility has to be taken care of constantly. By providing Joomla as a fully accessible foundation, as well as providing tools and information for accessibility of the standards and processes, we hope to make the Joomla ecosystem, as a whole, a beacon for accessibility in the digital world.

To achieve our aim of making Joomla completely accessible inside and out, we need your help. 
If you would like to volunteer or contribute to our team in any way, we would love to have you on board! Please join us or contact us on Volunteers Portal.

### How to report a problem or suggestion
If you find a bug in Joomla! CMS, please report it on the Issue Tracker. 
If you would like to report an issue about any of our *.joomla.org sites, please click on the "Report an issue" link located in the footer of that website.
Our dedicated volunteers will address any issues as soon as humanly possible.

### The Joomla! Accessibility Disclaimer
The Joomla Project is not responsible for compliance with the standards of accessibility of applications and extensions created with and/or for the Joomla CMS and Framework but is committed to making changes to the Joomla! CMS and Framework so that the available applications, extensions, and web sites have a solid foundation in keyboard support, focus support, ARIA markup and other assistive technologies.

**Comment by armenos** [comment]
If you guys are agree with this statement it can go online now and if we forgot something to mention please add your comment 
@mbabker @zwiastunsw @wilsonge @chmst @tarotray @uglyeoin @ylahav @moje-s @seenee 
@Joomv @jonrz @Sandra97 @ciar4n @dgt41 @brianteeman @coolcat-creations @C-Lodder @yvesh

**Comment by seenee** [comment]
There is no space betweeen first and second sentence in "Joomla! Accessibility Team (JAT)" part.

It's OK!!
let's just put it on-line 🥇

**Comment by chmst** [comment]
OK :-)

**Comment by ylahav** [comment]
ok...

**Comment by zwiastunsw** [comment]
Specify a date for the discussion, eg by 10 September, and then hand over the decision to the OSM Board. In my opinion it is okay.

**Comment by C-Lodder** [comment]
I wouldnt say that extension developers need to declare their level of a11y as they may not have any idea about it too.

I think this is too much to ask cause you will need to actually test the extension. If commercial extensions lie about their level of a11y for any reason, it may become an issue in countries where a11y compatible sites are the law.

**Comment by Sandra97** [comment]
Ok for me (without adding the extensions part as discussed together) 
I can add a date at the bottom of the article when published on jorg

**Comment by zwiastunsw** [comment]
@Sandra97 
> I can add a date... of the article when published on jorg
Good point. The statement should be dated!

**Comment by armenos** [comment]
@Sandra97 I think you can put this online, you are on board :) so you can pass to them if it's needed for their opinion :)

**Comment by armenos** [comment]
Without extension part

**Comment by zwiastunsw** [comment]
@armenos 
I think the opinion/decision of the OSM Board is necessary. This is a very important "political", directional statement.

**Comment by brianteeman** [comment]
push further should be progress further
in barrier-free way should be either in a barrier free way or free from barriers
while joomla core enables = while joomla enables

**Comment by brianteeman** [comment]
Joomla.org website and subdomains

this section probably should mention something about using non joomla tools that are beyond our control eg phpbb and mediawiki

**Comment by brianteeman** [comment]
journey.The accessibility   should be journey. Accessibility

**Comment by mbabker** [comment]
> this section probably should mention something about using non joomla tools that are beyond our control eg phpbb and mediawiki

The disclaimer at the end should cover this?

> committed to making changes to the Joomla! CMS and Framework so that

The Framework does not have any HTML producing utilities other than support for templating engines, so you don't need to include it in accessibility statements regarding improving the code because there is no code to change.

> authors of all extensions provided in the Directory will declare their level of accessibility

This mandate is IMO out of scope for an Accessibility Team decision.  And to be honest, it is probably something that cannot be reasonably tested or enforced as it relates to JED listings.  The only accessibility aspects an extension may cover are directly related to the default layouts which they provide, which may or may not be in use with the active site template (we use Akeeba Release System on https://downloads.joomla.org; even if its default set of layouts are accessible we have created overrides for most of those layouts therefore there is no guarantee that our layouts have the same level of accessibility).

**Comment by jeckodevelopment** [comment]
> authors of all extensions provided in the Directory will declare their level of accessibility

Agree with @mbabker on that. Out of scope for the Accessibility Team, even if it could be an interesting thing. And it could be a long term goal. But please remove it from the Accessibility Statement at this moment.

**Comment by Sandra97** [comment]
@jeckodevelopment As mentioned by Armen, the part regarding JED was mentioned here as information but was not intended to be merged in the jorg page. As this needs first to be disussed with JED to see what could be possible.

**Comment by zwiastunsw** [comment]
> This mandate is IMO out of scope for an Accessibility Team decision. 

Yes. But the Accessibility statement **is not a statement Joomla Accessibility Team**. This is a Joomla!Project statement. We do not decide, we propose a decision.
Also we understand that we must first agree with JED, how to do it. Armen wrote this clearly.
We can publish a statement without this excerpt, but we should not dismiss it. We can add it later, after discussion, if it really takes a long time. But I think that it can be agreed quickly, for example within a month.

> And to be honest, it is probably something that cannot be reasonably tested or enforced as it relates to JED listings. 

It would be great if we tested the extensions before. But this is unrealistic. And it is not necessary. This is to be a designer statement, ensuring that he has taken into account accessibility requirements.
Unfortunately, the main reason for barriers in the accessibility of software is ignorance, incompetence of designers. Ensuring accessibility is relatively easy at the design stage.
**Our goal is to draw designers' attention to this aspect of the production process**. Let's say: Take care of accessibility and tell us if your extension meets accessibility requirements.
If we do not mobilize extension designers and templates to accommodate accessibility requirements, **our work on the accessibility of Joomla will be wasted**.
So let's not say no to do that. Let's discuss how to do it to achieve the goal.

**Comment by armenos** [comment]
Guys can we publish, some parts of this document? And then we can make improvements.
I think the board must decide which parts are going online now.
We already have discussed how and what.
@Sandra97 Sandra can you put this partly online?

**Comment by jeckodevelopment** [comment]
@armenos Sandra will be available again in few days, Maybe ask @joomv

**Comment by Sandra97** [comment]
I'll be available in a few weeks ;)

**Comment by Sandra97** [comment]
[Accessibility statement](https://www.joomla.org/accessibility-statement.html) updated.
Do not hesitate to ask me for any edits.
